golang/cosmos/Makefile

```diff
@@ -1,5 +1,5 @@
-NAME := $(shell node -e 'console.log(require("./package.json").name)')
-VERSION := $(shell node -e 'console.log(require("./package.json").version)')
+NAME := $(shell sed -ne 's/.*"name": "\([^"]*\)".*/\1/p' package.json)
+VERSION := $(shell sed -ne 's/.*"version": "\([^"]*\)".*/\1/p' package.json)
```

Comment:
Filtering `package.json` with a regexp like this seems fragile.

Reply:
I left an explanatory comment.